packages/model/src/research-output.ts

```diff
@@ -404,3 +423,51 @@ export const convertBooleanToDecision = (bool?: boolean): DecisionOption => {
   }
   return bool ? 'Yes' : 'No';
 };
+
+export const toResearchOutputListItem = (
+  researchOutput: ResearchOutputResponse,
+): ResearchOutputListItem => {
+  const {
+    id,
+    published,
+    addedDate,
+    authors,
+    created,
+    documentType,
+    labs,
+    link,
+    teams,
+    title,
+    type,
+    workingGroups,
+    isInReview,
+    methods,
+    organisms,
+    environments,
+    subtype,
+    keywords,
+  } = researchOutput;
+
+  return {
+    id,
```

Comment:
I think we need to add "publishingEntity" otherwise it appears like this
<img width="1031" alt="Screenshot 2023-11-15 at 09 49 42" src="https://github.com/yldio/asap-hub/assets/16595804/c45575cc-6183-4a15-913f-0103b6c7e6b8">